assets/quick-add.css

```diff
@@ -332,3 +332,59 @@ quick-add-bulk .quantity__input {
   width: calc(6.5rem / var(--font-body-scale));
   flex-grow: 0;
 }
+
+.quick-add-bulk-no-variants__error {
+  position: absolute;
+  width: 95%;
+  height: auto;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  bottom: calc(100% + 0.5rem);
+  left: 50%;
+  transform: translate(-50%);
+  border: 1px solid rgb(var(--color-foreground));
+  border-radius: 5px;
+  background-color: rgb(var(--color-background));
+  z-index: 3;
+}
+
+.quick-add-bulk-no-variants__error ::before {
+  content: "";
+  position: absolute;
+  width: 20px;
+  height: 20px;
+  bottom: -18px;
+  left: 50%;
+  margin-left: -1rem;
+  border-width: 1rem;
+  border-style: solid;
+  border-color: rgb(var(--color-background)) transparent transparent transparent;
+  z-index: 2;
+}
+
+.quick-add-bulk-no-variants__error :after {
+  content: "";
+  position: absolute;
+  width: 20px;
+  height: 20px;
+  bottom: -20px;
+  left: 50%;
+  margin-left: -1rem;
+  border-width: 1rem;
+  border-style: solid;
+  border-color: rgb(var(--color-foreground)) transparent transparent transparent;
+  z-index: 1;
+}
+
+.quick-add-bulk-no-variants__error svg {
+  max-width: 1.5rem;
+  margin: 0 0.5rem;
+}
+
+.quick-add-bulk-no-variants__error-message {
+  padding: 0.5rem;
+  color: rgb(var(--color-foreground));
+}
+
```

Comment:
The font size feels a bit big here

https://screenshot.click/07-19-98s4t-w8ham.png